### ISSUE TITLE
On first focus, dropdown options are filling up the entire page

### ISSUE THREAD

**Issue by KhumboLihonga** [google/android-fhir#1757]
**Describe the bug**
When a dropdown is focused on for the first time, the options are displayed but they cover the entire page such that, the user cannot see what they are typing in order to filter the options.

**Component**
SDC library

**To Reproduce**
N/A

**Expected behavior**
I expect the options to be shown underneath the dropdown as is shown later in the video below.

**Screenshots**
https://user-images.githubusercontent.com/90775548/206134634-b033c143-2563-4e8e-bd6c-966c627d34af.mp4

**Smartphone (please complete the following information):**
- Device: Samsung A12
- Android version: 11
- Build number: RP1A.200720.012.A127FXXU4AUK1
- Version: 30

**Additional context**
Here is the [questionnaire](https://github.com/google/android-fhir/files/10174410/dropdown-sample.txt) used for the video above.

**Would you like to work on the issue?**
I think this can be a follow up to https://github.com/google/android-fhir/pull/1722

**Comment by shelaghm** [comment]
Yes, I agree with both of these issues and the proposed solutions.

1. dropdown options should not cover the input box. Options should be shown below the box.
2. all dropdown options should be visible when the user removes the typed text

**Comment by jingtang10** [comment]
Not enturely sure what we can do about 1) to be honest - i cannot reproduce this issue on my device. we don't really control the position of the dropdown, so the fact that it covers the input box is a little tricky to fix. any ideas @KhumboLihonga or @aditya-07?

about 2) we can probably do something about it.

**Comment by jingtang10** [comment]
i don't believe this is a P1 issue - the usability of the widget is not seriously impacted by this.

**Comment by KhumboLihonga** [comment]
@jingtang10 I've tried to position the dropdown in different parts of the questionnaire and I can see what you mean when you say you don't control the position of the dropdown. As per the [screenshot](https://user-images.githubusercontent.com/90775548/207038223-95e1754e-e82b-434a-85b0-4ba1827a7e0a.jpg), it's just not intuitive to the user to start typing when they see something like this when the dropdown first receives focus. However, I do agree that this is more of a minor issue as the widget itself works.
